devices/Mcp7940xx/Mcp7940xx.nuspec

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>nanoFramework.Iot.Device.Mcp7940xx</id>
+    <version>$version$</version>
+    <title>nanoFramework.Iot.Device.Mcp7940xx</title>
+    <authors>nanoFramework project contributors</authors>
+    <owners>nanoFramework,dotnetfoundation</owners>
```

Comment:
This one is deprecated. We're not using it anymore
``suggestion
```

Reply:
Did you mean the <owners> tag?